cw/tvla.py

```diff
@@ -856,11 +856,15 @@ def run_tvla(ctx: typer.Context):
 
             log.info("Plotting T-test Statistics vs. Number of Traces, this may take a while.")
             # Determine resolution.
-            xres_vec = [10e6, 1e6, 100e3, 10e3, 1e3]
+            xres_vec = [10e6, 1e6, 100e3, 10e3, 1e3, 100, 1]
```

Comment:
nit: out of curiosity why do we skip 1e2? I don't think this is really an issue, but on the first glance it looks like there is something missing ;)

Reply:
I think it's because the x-axis still looks nice when we have two-digit numbers. It's only with three-digit numbers that it begins to look cluttered.
Maybe we should remove `10e3` and `10e6` to be consistent.

Reply:
It's because I find the `M` and `k` suffices are very easy to interpret. I am always struggling with the 1e1, 10e1, 1e2 notation etc.

Also two and 3-digit labels are still kind of okayish to read. Let me add the `10` as well.

Reply:
Done.